src/decoders/novatel/src/rangecmp/range_decompressor.cpp

```diff
@@ -6,7 +32,7 @@ using namespace novatel::edie;
 using namespace novatel::edie::oem;
 
 //-----------------------------------------------------------------------
-//! Table used to expand the scaled Pseudorange STDs.  This is defined
+//! Table used to expand the scaled pseudo range STDs.  This is defined
```

Comment:
not super important, but I think pseudorange is the correct term here.

Reply:
fixed!